src/config/middlewares/authentication.js

```diff
@@ -0,0 +1,23 @@
+import jwt from 'jsonwebtoken';
+const jwtSecret = process.env.JWT_SECRET || 'SECRET';
+
+
+/**
+ * middleware for token validation
+ * @returns {void}
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Function} next
+ */
+export default function authenticationMiddleware(req, res, next) {
```

Comment:
authMiddleware seems like a shorter name, nice work

now you need to mount the middleware on games. userSearch routes
basically, only authenticated users should be able to save or search a user